mobilecoind-json/src/bin/main.rs

```diff
@@ -179,22 +179,21 @@ fn public_address(
 }
 
 /// Generates a request code with an optional value and memo
-#[post("/codes/request", format = "json", data = "<code_request>")]
-fn request_code(
+#[post("/codes/request", format = "json", data = "<request>")]
+fn get_request_code(
```

Comment:
I think this would be better named `create_request_code` or `generate_request_code`. `get_` and `read_` are too similar.

Reply:
yeah, it's also confusing to have request code requests running around... Here's what we have in the protobuf:

`    // b58 Codes
    rpc ReadRequestCode (ReadRequestCodeRequest) returns (ReadRequestCodeResponse) {}
    rpc GetRequestCode (GetRequestCodeRequest) returns (GetRequestCodeResponse) {}
    rpc ReadTransferCode (ReadTransferCodeRequest) returns (ReadTransferCodeResponse) {}
    rpc GetTransferCode (GetTransferCodeRequest) returns (GetTransferCodeResponse) {}
    rpc ReadAddressCode (ReadAddressCodeRequest) returns (ReadAddressCodeResponse) {}
    rpc GetAddressCode (GetAddressCodeRequest) returns (GetAddressCodeResponse) {}
`

Open to having different names in `mobilecoind-json` but this is where they originate

Reply:
Then that should be changed as well. Generate and Read seems good to me.

Reply:
I like parse... maybe Create/Parse? This will be a bit of work to correct as it's used in lots of places so I'll do it in a followup PR...